packages/connect-query/src/unary-hooks.test.ts

```diff
@@ -741,15 +743,15 @@ describe('unaryHooks', () => {
         >;
       });
 
-      it('passes through getPlaceholderData, when provided', () => {
+      it('passes through placeholderData, when provided', () => {
```

Comment:
Super nit: It's not really passing through placeholderData as much as it is running the right `getPlaceholderData` to generate the right placeholderData

Reply:
much appreciated! 0ac91b66868f00828712439ca13273d69e3968bb